src/mouse-keyboard-vr-display.js

```diff
@@ -161,6 +164,26 @@ MouseKeyboardVRDisplay.prototype.onMouseUp_ = function(e) {
   this.isDragging_ = false;
 };
 
+
+MouseKeyboardVRDisplay.prototype.requestPointerLock_ = function(element) {
+  // From pointerlock controls in three.js.
+  // - https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_pointerlock.html
+  element.requestPointerLock = element.requestPointerLock || element.mozRequestPointerLock || element.webkitRequestPointerLock;
+  if (/Firefox/i.test(navigator.userAgent)) {
+    var fullscreenchange = function(event) {
+      if (document.mozFullScreenElement === element) {
+        document.removeEventListener('mozfullscreenchange', fullscreenchange);
+        element.requestPointerLock();
+      }
+    };
+    document.addEventListener('mozfullscreenchange', fullscreenchange, false);
```

Comment:
chrome and firefox works on this. 

chrome support .requestPointerLock() even without fullscreen. While firefox requires to have fullscreen to have pointerlock. so on firefox, we first go fullscreen, then do the requestPointLock

so for chrome, it goes into the  ```else``` and does a normal element.requestPointerLock();